scipy/special/_basic.py

```diff
@@ -1337,9 +1337,10 @@ def riccati_jn(n, x):
 def riccati_yn(n, x):
     """Compute Ricatti-Bessel function of the second kind and its derivative.
 
-    The Ricatti-Bessel function of the second kind is defined as :math:`x
+    The Ricatti-Bessel function of the second kind is defined here as :math:`+x
     y_n(x)`, where :math:`y_n` is the spherical Bessel function of the second
-    kind of order :math:`n`.
+    kind of order :math:`n`. _Note that this is in contrast to a common convention
+    that includes a minus sign in the definition._
```

Comment:
The DLMF reference seems to support the current usage, I wasn't smart enough to read the older PDF reference. I guess the negation convention is related to the singularity.

Is the `_` notation doing what you intend in the rendered version?

![image](https://github.com/user-attachments/assets/bac82c76-db14-453d-9091-bbf4b348d678)

Reply:
No, I guess I was mixing up with markdown

Reply:
<img width="899" alt="image" src="https://github.com/user-attachments/assets/0aa30f4a-5822-4fb1-816e-13ac969bf8d2">

Thanks @tylerjereddy. Fixed.